### PR TITLE
Fix ruleset XML

### PR DESCRIPTION
**Pull request by szepeviktor**
From https://github.com/InteractionDesignFoundation/coding-standard/pull/1#pullrequestreview-1386151226

Mainly done with
`sed -i -e 's#^\(\s\+\)\(<.\+/>\)\(<!-- .\+ -->\)$#\1\3\n\1\2#' IxDFCodingStandard/ruleset.xml`

### CODE REVIEW

File: IxDFCodingStandard/ruleset.xml [hunk 1]
```diff
@@ -447,7 +442,7 @@
                 <element key="auth" value="null"/>
                 <element key="app" value="null"/>
                 <element key="resolve" value="null"/>
-                <element key="back" value="redirect()->back"/>
+                <element key="back" value="redirect()-&gt;back"/>
```
Comment (szepeviktor) [inline IxDFCodingStandard/ruleset.xml:548]: This was a syntax error.

**Review by alies-dev — APPROVED** [review]
great catch! Although IMO comments before rule/element are a bit less readable (too many focus on comments), it's better to automate such checks, so this is a good compromise. Thank a lot!

**Comment by szepeviktor** [comment]
> comments before rule/element are a bit less readable

That is not your opinion, that is a fact!
But XML is for machines only.

**Comment by szepeviktor** [comment]
I hope this standard will not change more frequently than once a quarter.

**Comment by alies-dev** [comment]
>I hope this standard will not change more frequently than once a quarter.

no, we used this ruleset for years for our main project (private repo), with very small changes all this time. By starting using it on other repositories we have find some issues, fix them and that's it.

**Pull request merged**